extensions/redis-cache/runtime/src/main/java/io/quarkus/cache/redis/runtime/RedisCacheRuntimeConfig.java

```diff
@@ -8,6 +8,13 @@
 
 @ConfigGroup
 public class RedisCacheRuntimeConfig {
+
+    /**
+     * The timeout for the compute method
+     */
+    @ConfigItem
+    public Optional<Duration> computeTimeout;
```

Comment:
If you're defining a default value in the `RedisCacheInfoBuilder`, why not set it here?

Reply:
Ah probably because you're checking `namedRuntimeConfig` and fallback to `defaultRuntimeConfig` before assuming the default value. If that's the case, ignore this comment

Reply:
Because the default value from the named cache would override the custom one from the default cache I think

Reply:
Yes precisely that